google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/readrows/ReadRowsRetryTest.java

```diff
@@ -355,6 +469,58 @@ RpcExpectation expectRequest(String... keys) {
       return this;
     }
 
+    RpcExpectation expectRequestForMultipleRowRanges(List<Range<String>> rowRanges) {
+      RowSet.Builder rowRange = requestBuilder.getRowsBuilder();
+      for (Range<String> range : rowRanges) {
+        rangeBuilder(range);
+      }
+      return this;
+    }
+
+    /**
+     * Build Row Range
+     *
+     * @param range
+     * @return
+     */
+    RowRange rangeBuilder(Range<String> range) {
```

Comment:
I'm a bit confused, whats this method for? you can use `Query.create(..).range(Range.ByteStringRange.create());` instead, which includes all the start / end open / close combinations https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Range.java#L69-L103